testing/obsolete

```diff
@@ -0,0 +1,9 @@
+# Usage:
```

Comment:
is this file needed for something?

Reply:
It is manual test for new code in function `Config::updateObsolete`. It is not connected to test suite. It could be connected latter.